shared/tsconfig.json

```diff
@@ -1,8 +1,6 @@
 {
   "extends": "../tsconfig-base.json",
   "compilerOptions": {
-    "outDir": "../../.tmp/tsbuildinfo/shared/",
```

Comment:
checking the diff on the `outDir` also revealed a minor bug here: this one was saving one directory too high. Check the parent directory of your lighthouse repo and there will be a `.tmp` in there, too. Sorry :)